soak-tests-common-sql/src/main/java/com/hazelcast/jet/tests/common/sql/TestRecordProducer.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.tests.common.sql;
+
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+
+public final class TestRecordProducer {
+
+    private TestRecordProducer() {
+    }
+
+    public static String produceTradeRecords(long itemsSubmitted, long itemsCountToSubmit, int timeInterval) {
```

Comment:
I don't think this method is implemented properly (maybe it has just incorrectly named parameters). For example based on parameters name I would expect that `produceTradeRecords(10, 5, 2);` will generate 5 items (because `itemsCountToSubmit=5`) but that doesn't happen, it generates `(TO_TIMESTAMP_TZ(10), 10), (TO_TIMESTAMP_TZ(12), 12), (TO_TIMESTAMP_TZ(14), 14)`. Also the name `itemsSubmitted` should be improved - without method implementation knowledge I would say it's number of items which have been already submitted but it's actually index from which it should start to generate items.

Another things: 
- if we run it with `itemsCountToSubmit=0` (or some negative number) it still generates one item.
- if `timeInterval` is 0 or lower it will never finish

We need to improve it (and probably add some javadoc) since it's util method.